onnxruntime/core/graph/model.h

```diff
@@ -199,14 +199,16 @@ class Model {
   static common::Status Save(Model& model, int fd);
 
   // Save the model to file using an external file for initializers larger than the given threshold (in bytes).
+  // external_file_name must be a UTF-8 string
   static common::Status SaveWithExternalInitializers(Model& model,
-                                                     const PathString& file_path,
+                                                     const std::filesystem::path& file_path,
```

Comment:
If I don't change this function's signature, when the functions in tensorprotoutils.cc call SaveWithExternalInitializers they will need to do an extra path to string conversation.  Later on I will do more cleanup on this file, For example, the "ToGraphProtoWithExternalInitializers" function in graph.h takes one string and one model path. I will change the std::string to std::filesystem::path. 
Internally every "std::filesystem::path" stores the string in native format, which means on Windows it will use a UTF-16 string and on Linux it will use a multibyte string.  As long as we don't convert between std::filesystem::path and std::string, there won't be any encoding issue. Therefore I suggest using std::filesystem::path for all path related things. But, I cannot change all places in one PR.